phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -115,7 +114,7 @@ public class ParameterizedIndexUpgradeToolIT extends BaseTest {
 
     @BeforeClass
     public static synchronized void saveTmp () throws Exception {
-        tmpDir = System.getProperty("java.io.tmpdir");
+        tmpPath = System.getProperty("java.io.tmpdir");
```

Comment:
This is the value of tmpDir before starting the minicluster.
It should only be used for restoring its value, and not be used in tests.

Reply:
could you elaborate on this please since all i did here was change the variable name?

Reply:
See the comment above.
the tmpdir values originally set, and re-set by minicluster must be kept separate.